packages/time-picker/src/common/picker.vue

```diff
@@ -223,7 +232,13 @@ export default defineComponent({
     }
     const emitInput = val => {
       if (!valueEquals(props.modelValue, val)) {
-        ctx.emit('update:modelValue', val)
+        let formatValue
+        if (Array.isArray(val)) {
+          formatValue = val.map(_ => formatter(_, props.valueFormat))
+        } else if(val) {
+          formatValue = formatter(val, props.valueFormat)
+        }
+        ctx.emit('update:modelValue', val ? formatValue : val)
```

Comment:
ctx.emit('update:modelValue', val || formatValue)

Reply:
ctx.emit('update:modelValue', formatValue || val)

Reply:
what I mean is to simplify it from `val ? formatValue : val` to `val || formatValue`

Reply:
when empty case, I did't change the origin way

Reply:
for example：click clear button

Reply:
actually  `ctx.emit('update:modelValue', val ? formatValue : val)` is the right way